src/rpc_client/src/meta_client.rs

```diff
@@ -1028,11 +1030,24 @@ pub enum MetaAddressStrategy {
     List(Vec<String>),
 }
 
+impl fmt::Display for MetaAddressStrategy {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        match self {
+            MetaAddressStrategy::LoadBalance(addr) => {
+                write!(f, "LoadBalance({})", addr)?;
+            }
+            MetaAddressStrategy::List(addrs) => {
+                write!(f, "List({:?})", addrs)?;
+            }
+        }
+        Ok(())
+    }
+}
+
 type MetaMemberClient = MetaMemberServiceClient<Channel>;
 
 struct MetaMemberGroup {
-    client_cache: HashMap<String, MetaMemberClient>,
-    members: HashSet<String>,
+    members: LruCache<String, Option<MetaMemberClient>>,
```

Comment:
I guess using `moka` is better here. 🥵 `lru-rs` is designed and tweaked for executor cache, and `moka` should be more friendly to `async` and concurrency.

Reply:
🥵  I see. Will change it.